app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/MainNavHost.kt

```diff
@@ -19,30 +19,44 @@
 package com.infomaniak.swisstransfer.ui.screen.main
 
 import androidx.compose.foundation.layout.safeDrawingPadding
-import androidx.compose.material3.Text
+import androidx.compose.material3.adaptive.WindowAdaptiveInfo
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.navigation.NavHostController
 import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
 import androidx.navigation.toRoute
 import com.infomaniak.swisstransfer.ui.navigation.MainNavigation.*
+import com.infomaniak.swisstransfer.ui.screen.main.received.ReceivedScreen
+import com.infomaniak.swisstransfer.ui.screen.main.sent.SentScreen
+import com.infomaniak.swisstransfer.ui.screen.main.settings.SettingsScreenWrapper
+import com.infomaniak.swisstransfer.ui.screen.main.transferdetails.TransferDetailsScreen
 
 @Composable
-fun MainNavHost(navController: NavHostController, startDestination: SentDestination) {
+fun MainNavHost(
+    navController: NavHostController,
+    startDestination: SentDestination,
+    windowAdaptiveInfo: WindowAdaptiveInfo,
```

Comment:
Do we always need to pass down the WindowAdaptiveInfo instance through all functions? Is there no way to get access it globally inside whichever function needs it?